allensdk/brain_observatory/behavior/behavior_project_cache/external/behavior_project_metadata_writer.py

```diff
@@ -0,0 +1,320 @@
+import argparse
+import json
+import logging
+import os
+from typing import Union
+
+import pandas as pd
+
+import allensdk
+from allensdk.brain_observatory.behavior.behavior_project_cache import \
+    BehaviorProjectCache
+from allensdk.brain_observatory.behavior.behavior_project_cache.tables \
+    .experiments_table import \
+    ExperimentsTable
+from allensdk.brain_observatory.behavior.behavior_project_cache.tables \
+    .ophys_sessions_table import \
+    BehaviorOphysSessionsTable
+from allensdk.brain_observatory.behavior.behavior_project_cache.tables \
+    .sessions_table import \
+    SessionsTable
+
+
+class BehaviorProjectMetadataWriter:
+    """Class to write project-level metadata to csv"""
+
+    def __init__(self, behavior_project_cache: BehaviorProjectCache,
+                 out_dir: str, project_name: str):
+        self._behavior_project_cache = behavior_project_cache
+        self._out_dir = out_dir
+        self._project_name = project_name
```

Comment:
The methods and manipulations in the class make me think this might be specific to the `visual-behavior-ophys` project, do you think this class might be used for projects like `visual-behavior-neuropixels` too?